linera-core/src/node.rs

```diff
@@ -194,6 +194,9 @@ pub enum NodeError {
     #[error("Worker error: {error}")]
     WorkerError { error: String },
 
+    #[error("Missing certificates: {error}")]
+    ReadCertificatesError { error: String },
```

Comment:
Let's not introduce error variants that are only created in tests. You could just use `MissingCertificateValue` in the test.

Reply:
Ok.